tests/commonUtils.sh

```diff
@@ -15,8 +15,8 @@
 #   limitations under the License.
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-RUNTIME_IMAGE='docker.io/istio/runtime:2017-03-22-21.21.48'
-
+MANAGER_HUB_TAG='docker.io/istio,2017-03-22-21.21.48'
```

Comment:
Yes that s correct. Reading from istio.VERSION is what you want. But don't change the format without changing the update script. The reason I am using the full version is because the script goes over all the file in the repo and update them. I like the idea of providing a tag for manager and generate all other images name from that.